apps/sap-commerce-cloud/src/AppConfig/AppConfig.spec.tsx

```diff
@@ -2,15 +2,16 @@ import React from 'react';
 import { render } from '@testing-library/react';
 import { ParameterDefinition } from '../interfaces';
 import AppConfig from './AppConfig';
+import { vi } from 'vitest';
```

Comment:
shoutout to @david-shibley-contentful for figuring out that we needed to replace jest with vi 😄